site/docs/v1/tech/installation-guide/cloud.adoc

```diff
@@ -203,46 +203,87 @@ image::installation-guides/cloud/deployments-manage.png[Manage deployment,width=
 === Upgrading a Deployment
 
 If your deployment is not running the latest version of FusionAuth, you may upgrade it.
+It is a good idea to run the latest released version of FusionAuth, which has the latest bug fixes, security updates and features.
+
+However, you will not be forced to upgrade on a certain schedule. 
 
 [NOTE]
 ====
-Upgrade management is only available if the deployment is not currently running the latest available FusionAuth version. 
+Upgrade management is only available if the deployment is not currently running the latest released FusionAuth version. 
 ====
 
-There will be downtime of between 5 minutes and 60 minutes. The exact downtime duration depends on the type of deployment, amount of data in your system, and database changes required by the version upgrade. Consult the link:/docs/v1/tech/release-notes[relevant release notes] for functional changes as well.
+==== Upgrade Duration
+
+This upgrade process takes between 5 minutes and 60 minutes of time.
+The exact duration of system downtime depends on the type of deployment, amount of data in your system, and database changes required by the version upgrade.
+
+.Upgrade Durations in FusionAuth Cloud
+[cols="1,1,1"]
+|===
+| Hosting Option | Duration of Upgrade | Expected Downtime 
+
+|Basic Cloud
+|5 to 60 minutes
+|5 to 60 minutes
+
+|Business Cloud
+|5 to 60 minutes
+|5 to 60 minutes
 
-Due to the downtime, it is recommended that you schedule the upgrade for a low traffic period. Test the upgrade process on development or test servers first.
+|HA Cloud
+|5 to 60 minutes
+|seconds
```

Comment:
> Short-lived duration (less than 60 seconds)

Reply:
That's not what I heard Brian say :)

Reply:
Confirmed with Brian that it is seconds of downtime for user logins